app/main.py

```diff
@@ -1,8 +1,35 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.wife = None
+        self.husband = None
+
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    persons = [Person(data["name"], data["age"]) for data in people]
```

Comment:
The list comprehension correctly creates `Person` objects from the provided list of dictionaries. Ensure that each dictionary in `people` contains the required keys `name` and `age` to avoid potential KeyErrors.